src/components/RouteForm.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import { Button, Form, Container, Message } from "semantic-ui-react";
+import getRoute from "../modules/route_request";
+
+const RouteForm = () => {
+  const [routeInformation, setRouteInformation] = useState();
+  const [invalidLocationMessage, setInvalidLocationMessage] = useState("");
+
+  const createRoute = async (event) => {
+    event.preventDefault();
+
+    const from = event.target.origin.value;
+    const to = event.target.destination.value;
+    const response = await getRoute.create(from, to)
```

Comment:
getRoute.create? is it a get request or a post request? 🤔

Reply:
it's a post request, we just changed the naming for getRoute to calculateRoute.